service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/SubmittedCaseEventHandlerTest.java

```diff
@@ -131,27 +131,12 @@ void init() {
             when(mapper.convertValue(any(), eq(CaseData.class))).thenReturn(caseData);
         }
 
-        @Test
-        void shouldNotPayIfPaymentIsToggledOff() {
-            final CallbackRequest request = callbackRequest(OPEN);
-
-            final SubmittedCaseEvent submittedCaseEvent = new SubmittedCaseEvent(request);
-
-            when(featureToggleService.isPaymentsEnabled()).thenReturn(false);
-
-            submittedCaseEventHandler.makePayment(submittedCaseEvent);
-
-            verifyNoMoreInteractions(paymentService, applicationEventPublisher);
-        }
-
         @ParameterizedTest
         @EnumSource(value = State.class, mode = EXCLUDE, names = {"OPEN"})
         void shouldNotPayIfCaseStateIsDifferentThanOpen(State state) {
```

Comment:
Not your test but I feel like this test is testing unncessary things.
Unless I am remembering things wrong the code this test is calling will only be called from the `OPEN` or `RETURNED` state.
I feel like testing it on all other states is a waste of time.

Reply:
Also the method towards the bottom has either a redundant variable or is not passing its variable to the overloaded method correctly 
```java
    private static CallbackRequest callbackRequest(State state) {
        return callbackRequest(OPEN, emptyMap());
    }
```

Reply:
I see why you put both comments here :). I think, it's okay for test to cover all states like black box testing. Let me know what you think.

For parameterised test to work, i need above method as is, i guess that's the reason you put both comments here?

Reply:
>I think, it's okay for test to cover all states like black box testing. Let me know what you think.

sure makes sense.

>For parameterised test to work, i need above method as is.

The state variable isn't used and as such can be removed

Reply:
done